qtfred/src/ui/BriefingEditorDialog.h

```diff
@@ -0,0 +1,22 @@
+#ifndef BRIEFINGEDITORDIALOG_H
+#define BRIEFINGEDITORDIALOG_H
+
+#include <QDialog>
+
+namespace Ui {
+class BriefingEditorDialog;
+}
+
+class BriefingEditorDialog : public QDialog
```

Comment:
Dialogs will be put into the `fso::fred::dialogs` namespace so that should also be done here. You can make the UI compiler generate the class with the appropriate namespace by specifying the name of the class with the full namespace (see FredView.ui for an example).